components/eam/src/physics/cam/cloud_cover_diags.F90

```diff
@@ -35,7 +35,7 @@ subroutine cloud_cover_diags_init(sampling_seq)
   call phys_getopts(history_amwg_out           = history_amwg  )
  
   if (history_amwg) then
-      call add_default ('CLOUD   ', 1, ' ')
+!      call add_default ('CLOUD   ', 1, ' ')
```

Comment:
this could also probably be deleted rather than commented - we get the same output from the new loc of CLOUD outfld so will never go back to this old ver.